tests/test_graphical_units/test_last_frame.py

```diff
@@ -1,31 +1,27 @@
 import pytest
 
 import manim as mn
+from manim import color as C
 from ..utils.testing_utils import get_scenes_to_test
 from ..utils.GraphicalUnitTester import GraphicalUnitTester
 
-mn.config.frame_rate = 5
-mn.config.pixel_height = 150
-mn.config.pixel_width = 150
-
-
-def make_lbl(num):
-    text = mn.Text(f"{num}").scale(7)
-    text.num = num
-    return text
-
-
-def update_lbl(lbl, dt):
-    lbl.num += 1
-    lbl.become(make_lbl(lbl.num))
-
 
 class LastFrame(mn.Scene):
```

Comment:
**Comment/issue :** I'm worry that it won't test anything, as graphical unit tests are run with the equivalent of `-s` flag ( every animation is skipped)
therfore, the last t value is always the run_time.

Reply:
I'm not sure how this tests the change either...

Reply:
I was going through older issues/PRs again and recall @kolibril13 mentioning a way to test video tests which is VERY applicable here as a way to test this PR. #1035 

I recommend we test in this manner.

Reply:
for the record, in #1019 I Implemented such tests on t values, so no need to implement this here.